src/main/java/seedu/address/logic/commands/Command.java

```diff
@@ -1,20 +1,8 @@
 package seedu.address.logic.commands;
 
-import seedu.address.logic.commands.exceptions.CommandException;
-import seedu.address.model.Model;
-
 /**
- * Represents a command with hidden internal logic and the ability to be executed.
+ * Represents a command with hidden internal logic and the ability to be executed. Exists for polymorphism purposes.
  */
```

Comment:
Could we store some common attributes here instead of making `Command` an empty base class?

Reply:
Hmm.... Currently, it seems like there aren't really any common attributes we can store in `Command`. Maybe in the future, if we find some common attribute we could store, we'll add it in.